darts/models/torch_forecasting_model.py

```diff
@@ -754,6 +754,9 @@ def _sample_tiling(self, input_data_tuple, batch_sample_size):
                 tiled_input_data.append(None)
         return tuple(tiled_input_data)
 
+    def _batch_to_device(self, batch):
+        return [elem.to(self.device) if isinstance(elem, torch.Tensor) else elem for elem in batch]
```

Comment:
You return a list and not a tuple? A tuple might be safer as it doesn't change the type for downstream tasks.